src/app/portlets/content-types/fields/content-type-fields-properties-form/field-properties/checkbox-property/checkbox-property.component.ts

```diff
@@ -33,7 +36,37 @@ export class CheckboxPropertyComponent extends BaseComponent {
         );
     }
 
+    ngOnInit(): void {
+        this.indexedCheckbox = this.group.controls.indexed;
```

Comment:
You don't this this either, you can access the ```FormControl``` but it's better to use the ```.get``` like: ```this.group.get('indexed')```.